Website/Packages/datax-pipeline/src/modules/flowDefinition/components/flowDefinitionPanel.jsx

```diff
@@ -318,7 +351,42 @@ class FlowDefinitionPanel extends React.Component {
         );
     }
 
+    renderScheduleTab() {
+        if (this.props.input.mode === Models.inputModeEnum.streaming){
+            return null;
+        }
```

Comment:
can you make sure you open the datax-pipeline folder in VS Code and save any open files to get Prettier to kick in to help you style our codebase consistently. I can tell by looking at the code where this should have looked like "} else {" automatically. I assume you opened the Website\Packages folder which is why you didn't get auto formatting on save operations. If you want to open the parent folder like this, try adding a .prettierrc file at that folder level to solve this going forward.

Reply:
i.e. add .prettierrc file to Website\Packages folder. I haven't tried it so not sure if it will work but give it a try. You should look online for a solution, I'm sure its supported in some way.

Reply:
Right. I have to open Packages\datax-pipleline folder.  I tried to add .prettierrc file to Website\Packages folder but it didn't work.